### PR TITLE
Responding to Ofir-san’s review comments　

### PR DESCRIPTION
**Pull request by KazunoriSumiya**
## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tests_pytest/common_tests/unit_tests/test_model_collector.py [hunk 1]
```diff
@@ -68,13 +69,46 @@ def test_create_stats_collector_for_node_activation_disabled(self, fw_info_mock)
...
+        """
+        # Set up a fake node with activation quantization enabled and prior_info attributes.
+        node = Mock()
+        node.is_activation_quantization_enabled.return_value = False
+        node.is_fln_quantization.return_value = True
+        node.type = DummyLayer
+        node.prior_info = None
+
+        collector = create_stats_collector_for_node(node, fw_info_mock)
+        assert isinstance(collector, StatsCollector)
```
Comment (kkawa14) [inline tests_pytest/common_tests/unit_tests/test_model_collector.py:110]: node.prior_info = Mock(min_output=None, max_output=None)
についてのテストを追加
後、collector.mpcc.init_min_value、collector.mpcc.init_max_valueがNoneにセットされているかのAssert確認を追加

File: tests_pytest/common_tests/unit_tests/test_model_collector.py [hunk 2]
```diff
@@ -237,9 +325,104 @@ def test_infer_without_hessian(self, fw_impl_mock, fw_info_mock):
...
+        self.fake_hessian_result = np.random.randn(*input_shape)
+        self.fake_hessian_info_service.fetch_hessian = Mock(
+            wraps=self.fake_hessian_info_service.fetch_hessian,
+            return_value={'node1': self.fake_hessian_result, 'node2': self.fake_hessian_result}
+        )
+
+        self.qc = DEFAULTCONFIG
+        self.infer_input = [np.random.randn(*input_shape)]
+
+    def test_infer_without_hessian_in_FLN(self, fw_impl_mock, fw_info_mock):
```
[the remark this answers is not in the document]
Comment (KazunoriSumiya) [reply tests_pytest/common_tests/unit_tests/test_model_collector.py]: 関数名を変更しました。

File: tests_pytest/common_tests/unit_tests/test_model_collector.py [hunk 3]
```diff
@@ -237,9 +325,104 @@ def test_infer_without_hessian(self, fw_impl_mock, fw_info_mock):
...
+        self.qc.activation_error_method = QuantizationErrorMethod.MSE
+        mc = ModelCollector(self.graph, fw_impl_mock, fw_info_mock, qc=self.qc, hessian_info_service=self.fake_hessian_info_service)
+
+        mc.infer(self.infer_input)
+        # Check that inference runs without requiring gradients.
+        fw_impl_mock.run_model_inference.assert_called_once_with(mc.model, self.infer_input, requires_grad=False)
+        # Confirm that the Hessian service is not used.
+        mc.hessian_service.fetch_hessian.assert_not_called()
+
+    def test_infer_with_hessian_in_FLN(self, fw_impl_mock, fw_info_mock):
```
[the remark this answers is not in the document]
Comment (KazunoriSumiya) [reply tests_pytest/common_tests/unit_tests/test_model_collector.py]: 関数名を変更しました。

File: tests_pytest/common_tests/unit_tests/test_model_collector.py [hunk 4]
```diff
@@ -250,9 +433,9 @@ def test_infer_with_hessian(self, fw_impl_mock, fw_info_mock):
         # Confirm that the Hessian data is fetched.
         mc.hessian_service.fetch_hessian.assert_called_once()
 
-    def test_update_statistics_called(self, fw_impl_mock, fw_info_mock):
+    def test_update_statistics_called_in_FLN(self, fw_impl_mock, fw_info_mock):
```
[the remark this answers is not in the document]
Comment (KazunoriSumiya) [reply tests_pytest/common_tests/unit_tests/test_model_collector.py]: 関数名を変更しました。

**Review by kkawa14 — APPROVED** [review]
